sources/main.cpp

```diff
@@ -31,23 +31,32 @@
 #include "DYNMarginCalculationLauncher.h"
 #include "DYNComputeLoadVariationLauncher.h"
 #include "DYNComputeSimulationLauncher.h"
+#include "DYNCriticalTimeLauncher.h"
 
 using DYNAlgorithms::ComputeSimulationLauncher;
 using DYNAlgorithms::SystematicAnalysisLauncher;
 using DYNAlgorithms::MarginCalculationLauncher;
 using DYNAlgorithms::ComputeLoadVariationLauncher;
+using DYNAlgorithms::CriticalTimeLauncher;
 
 namespace po = boost::program_options;
 
 static void launchSimulation(const std::string& jobFile, const std::string& outputFile);
 static void launchMarginCalculation(const std::string& inputFile, const std::string& outputFile, const std::string& directory, int nbThreads);
 static void launchSystematicAnalysis(const std::string& inputFile, const std::string& outputFile, const std::string& directory, int nbThreads);
 static void launchLoadVariationCalculation(const std::string& inputFile, const std::string& outputFile, const std::string& directory, int variation);
+static void launchCriticalTimeCalculation(const std::string& inputFile, const std::string& outputFile, const std::string& directory, int nbThreads);
 
 int main(int argc, char** argv) {
   std::string simulationType = "";
   std::string inputFile = "";
   std::string outputFile = "";
+
+  std::string simulationMC_SA_CS = "Set the simulation type to launch : MC (Margin calculation), SA (systematic analysis), CS (compute simulation)";
+  std::string simulationCTC = " or CTC (critical time calculation)";
+  std::string allSimulationsTypeStr = simulationMC_SA_CS.append(simulationCTC);
```

Comment:
Why declaring this string in two steps?

Reply:
I didn't know how to declare a literal string on multilines. Si I did this... I fixed it.